mw-webapp/src/component/header/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import {NavLink} from "src/component/link/Link";
+import styles from "src/component/header/Header.module.scss";
+import {useAuth} from "src/utils/useAuth";
+import {Button} from "src/component/button/Button";
+import {pages} from "src/router/pages";
+
+const handleGoogleSignIn = useAuth();
+const buttonValue = "Sign in with Google";
+const linkValue = "Workflow";
+const logo = "master-way";
+
+
+export const Header = () => {
+  return (
+    <div className={styles.header}>
+      <div className={styles.title}>
+        {logo.toUpperCase()}
```

Comment:
Usually if logo is text we can use h1. What do you think is it a good idea?

Reply:
Done.